MeshLib/CoordinateSystem.cpp

```diff
@@ -31,4 +30,23 @@ CoordinateSystem::CoordinateSystem(const Element &ele)
     }
 }
 
+unsigned char CoordinateSystem::getCoordinateSystem(const GeoLib::AABB &bbox) const
+{
+    unsigned char coords = 0;
+
+    const MathLib::Vector3 pt_diff(bbox.getMinPoint(), bbox.getMaxPoint());
+
+    // The axis aligned bounding box is a from the right half-open interval.
+    // Therefore, the difference between the particular coordinates of the
+    // points is modified by the unit in the last place towards zero.
+    if (std::nexttoward(std::abs(pt_diff[0]), 0.0) > .0)
```

Comment:
Are there any implications from those tests, e.g., if something associated with a 3D model (e.g. a surface) happens to lie in a coordinate plane (e.g. x-y plane)? (I don't know, where this method is used, so maybe it's never will be a problem.)